open-api/rest-catalog-open-api.yaml

```diff
@@ -1581,6 +1608,17 @@ components:
         type: string
       example: [ "accounting", "tax" ]
 
+    PageToken:
+      description:
+        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables). 
+        Clients will initiate the first paginated request by sending an empty `pageToken` e.g. `GET /tables?pageToken` or `GET /tables?pageToken=`
+        signaling to the service that the response should be paginated.
+
+        Servers that support pagination will recognize `pageToken` and return a `next-page-token` in response if there are more results available.
+        After the initial request, it is expected that the value of `next-page-token` from the last response is used in the subsequent request.
+        Servers that do not support pagination will ignore `next-page-token` and return all results.
```

Comment:
Servers that do not support pagination will ignore `pageToken` right? This should be the query param not the response field.

Reply:
Will fix this.